### DISCUSSION TITLE
Tracking/logs of queries by user?

### DISCUSSION THREAD

**Discussion by jjkonway** [ciur/papermerge#255] [Q&A] [answered]
Hello im new to Papermerge, it seems really cool . Is there a method to save the users queries in the search bar? Or suggested way of implementing it? 

Thanks

**Comment by ciur** [comment, marked as answer]
Hi @jjkonway,

No there is no such functionality yet. One way to implement it is to remember on client side  (using document storage) and providing user suggestions of previous queries. 
As I already said - it is not implemented it.
But It would be a really cool feature. I will definitely implement it in future versions of Papermerge.
